native/packages/react-native-bpk-component-pagination-dots/package.json

```diff
@@ -15,6 +15,7 @@
     "react-native": ">= 0.47.0"
   },
   "dependencies": {
+    "@skyscanner/react-native-transitiongroup": "^1.1.1",
```

Comment:
A new dependency -> https://github.com/madsleejensen/react-native-transitiongroup/

Based on the web version https://reactcommunity.org/react-transition-group/, this is what controls the animation of items entering and leaving the react tree. We had to fork it to fix a but with ordering (we have follow up tasks to address this).